### PR TITLE
Only check for the Java binary lazily

### PR DESCRIPTION
**Pull request by zommerfelds**
Following up on my comment in https://github.com/Ribeiro-Tiago/bundletool/pull/19

### CODE REVIEW

**Comment by zommerfelds** [comment]
Sorry about that!

File: src/index.ts [hunk 1]
```diff
@@ -4,10 +4,18 @@ import commandExists from 'command-exists';
 import { Command } from "./types";
 import { BUNDLETOOL_FILE_PATH } from "../constants";
 
-if (!commandExists.sync('java')) {
-  throw new Error("Java is required for bundletool");
+
+var checkedJava = false;
```
Comment (Ribeiro-Tiago) [inline src/index.ts:8]: use let instead

**Comment by Ribeiro-Tiago** [comment]
No worries, I didn't even think about so that good catch :D

**Comment by zommerfelds** [comment]
Done!

**Pull request merged**